pages/basic_concepts/cache_configuration.md

```diff
@@ -75,6 +75,14 @@ contained a **dirty data** before switching to *Pass-Through* mode, then
 from **cache storage** until all the cache lines will be
 [**cleaned**](/cache_operations.html).
 
+### Write-Only
+In *Write-Only* mode, the **cache engine** writes the data exactly like
+in *Write-Back* mode and still there is a risk of data loss
```

Comment:
I'd rephrase it to first explain how it works (for both read and write), then describe the benefits (write speed increase), and at the very end inform about the consequences (reads not accelerated, potential data loss).

Reply:
Rephrased, but waiting for next review.